### PR TITLE
Wording.

### PR DESCRIPTION
**Pull request by smoe**
A continuation of https://github.com/LinuxCNC/mesaflash/pull/56. I had found a few typos while rereading this for the README.source for the linuxcnc packaging. Sorry and thanks!

### CODE REVIEW

File: debian/README.source [hunk 1]
```diff
@@ -2,12 +2,11 @@ Debian packaging of LinuxCNC/Mesaflash
 ======================================
 
 The following steps were performed to transform Upstream's git repository
-into a source tarball and a debian subdirectory that shines for the main
-distribution. The major obstacle is that Mesaflash is used across a wide
-span of distributions, from Wheezy  (Jessie anyon?) up to the latest
-Ubuntu releases. Not too many changes are required for Debian unstable,
-the most prominent one likely is that there is already a debiah directory
-in the source tree.
+into a source tarball and to provide a debian subdirectory that shines
```
Comment (jepler) [inline debian/README.source:5]: I'm not sure what "shines" means in this context.

Comment (jepler) [reply debian/README.source:5]: bump

Comment (smoe) [reply debian/README.source:5]: "looks very good" - let me eliminate this.

**Comment by smoe** [comment]
This PR continues at https://github.com/LinuxCNC/mesaflash/pull/63 .

**Pull request closed without merging**